training/culture-and-process/CULTURE.md

```diff
@@ -0,0 +1,48 @@
+Culture in DP
+===========================
+Our culture and how we work is important for us to create an inclusive, empathetic and successful environment that works for everybody. 
+
+
+### Principles
+
+#### Inclusion
+We believe people from different backgrounds, with different identities and experiences, make our product, and our team, better. 
+
+We know a happy, comfortable and safe environment where our people are free to express themselves and their ideas means we create the best experiences for our users. 
+
+#### User first
+We put out users at the heart of everything we do. This includes external and internal users. We know our users come from everywhere with a range of expertise and needs which helps to create a better product for all.
+
+We follow [GDS design principles](https://www.gov.uk/guidance/government-design-principles#start-with-user-needs). 
+
+#### Open communication and collaboration
+We code in the open. This means most of our code is publicly available on Github and it hosts a lot of our coding discussions.
+
+We encourage as many conversations and calls as possible to happen in open slack channels so that they are inclusive to all and can act as a source of information in the future. To let others know your concentrating on a task or that you'd like to minimise distractions use the headphones emoji 🎧 .
+
+
+Whichever platform we use to communicate we use open and inclusive language. 
+
+#### Self care
+We believe in everyone being happy and well and we know that everyone has different personal needs, circumstances and responsibilities. This means healthy and flexible working patterns and being mindful of others availability when scheduling events and communicating.
```

Comment:
SPAG - "others' availability"

Perhaps a slight reword to break the sentence structure a little:

"healthy and flexible working patterns, as well as being mindful..."

Reply:
👍

Reply:
👍